specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json

```diff
@@ -437,26 +437,432 @@
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/usages": {
+      "get": {
+        "operationId": "DatabaseAccounts_ListUsages",
+        "x-ms-examples": {
+          "CosmosDBDatabaseAccountGetUsages": { "$ref": "./examples/CosmosDBDatabaseAccountGetUsages.json" }
+        },
+        "description": "Retrieves the usages (most recent data) for the given database account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/usageFilterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The usages for the database account were retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/UsagesResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/databases/{databaseRid}/usages": {
+      "get": {
+        "operationId": "Database_ListUsages",
+        "x-ms-examples": {
+          "CosmosDBDatabaseGetUsages": { "$ref": "./examples/CosmosDBDatabaseGetUsages.json" }
+        },
+        "description": "Retrieves the usages (most recent data) for the given database account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseRidParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/usageFilterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The usages for the database were retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/UsagesResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/databases/{databaseRid}/collections/{collectionRid}/usages": {
+      "get": {
+        "operationId": "Collection_ListUsages",
+        "x-ms-examples": {
+          "CosmosDBCollectionGetUsages": { "$ref": "./examples/CosmosDBCollectionGetUsages.json" }
+        },
+        "description": "Retrieves the usages (most recent storage data) for the given database account.",
```

Comment:
description identical to `DatabaseAccounts_ListUsages`

Reply:
fixed